infra/Makefile

```
@@ -1,5 +1,9 @@
 .PHONY: help migrate-up migrate-down migrate-status docker-up docker-down clean docker-prod-up generate-secrets setup-env-files migrate-force
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)